cookbook/bundles/dependencies.rst

```diff
@@ -0,0 +1,52 @@
+.. index::
+   single: Bundle; Dependencies
+
+How to Use Bundle Dependencies to load other Bundles
+====================================================
+
+.. versionadded:: 2.7
+    Bundle dependencies was introduced in Symfony 2.7.
+
+When working on your own bundle(s), you'll sometimes have the need to reuse
+other bundles, either by just requiring, overriding or inherit from them.
+
+While Composer takes care about making sure these dependencies are loaded,
+you'll also need to enable these bundles in the kernel.
+
+If your bundle is meant to be reused, bundle dependencies will complicate
+your installation documentation. This makes installation and upgrading your
+bundle more tedious for your users.
+
+You can avoid this by specifying your dependencies, this will make sure
+they are loaded, and *before* your own bundle to make sure you can extend
```

Comment:
fixed, with one twist. I did not end the sentence with "bundle", but instead "them" as the subject is the dependencies so that felt clearer, might be even more clearer by consistently saying *the dependencies".  ok?